mcp/aruuznigar.py

```
@@ -0,0 +1,424 @@
+    Returns:
+        A dict with 'result' verdict, 'distance_from_own_bahr',
+        'distance_from_reference_bahr', and full 'reference_misra' / 'misra'
+        detail blocks.
+    """
+    line1 = _format_islah_for_compare(_post("/api/islah", {"text": reference_misra}))
+    line2 = _format_islah_for_compare(_post("/api/islah", {"text": misra}))
+    return _format_compare_result(line1, line2)
+
+
+@mcp.tool()
+def help() -> str:
```

Ruff (0.15.13)

[error] Variable `help` is shadowing a Python builtin

(A001)